Enable build unit test (using catch), and enable HIT test simultanouesly
Index: hipamd-rocm-5.6.0/CMakeLists.txt
===================================================================
--- hipamd-rocm-5.6.0.orig/CMakeLists.txt
+++ hipamd-rocm-5.6.0/CMakeLists.txt
@@ -447,8 +447,10 @@ if(${RUN_HIT} EQUAL 0)
     execute_process(COMMAND "${CMAKE_COMMAND}" -E copy_directory "${HIP_COMMON_BIN_DIR}" "${HIP_ROOT_DIR}/bin" RESULT_VARIABLE RUN_HIT ERROR_QUIET)
 endif()
 if(HIP_CATCH_TEST EQUAL "1")
-	message(STATUS "Building of catch tests through hipamd is no longer supported. Testing targets will not be available. catch tests have been moved to an independent github project hip-tests. Please refer to hip-tests Readme for build instructions! ")
-else()
+    enable_testing()
+    set(HIP_PATH ${HIP_ROOT_DIR})
+    add_subdirectory(${HIP_COMMON_DIR}/tests/catch ${PROJECT_BINARY_DIR}/catch)
+endif()
     if(${RUN_HIT} EQUAL 0)
         set(CMAKE_MODULE_PATH "${HIP_ROOT_DIR}/cmake" ${CMAKE_MODULE_PATH})
         include(${HIP_COMMON_DIR}/tests/hit/HIT.cmake)
@@ -457,7 +459,6 @@ else()
         message(STATUS "Testing targets will not be available. To enable them please ensure that the HIP installation directory is writeable. Use -DCMAKE_INSTALL_PREFIX to specify a suitable location")
     endif()
 endif()
-endif()
 
 #############################
 # Code analysis
